WordPress/UITests/Tests/NotificationTests.swift

```diff
@@ -44,4 +44,15 @@ class NotificationTests: XCTestCase {
             .replyToComment(withText: .commentText)
             .verifyReplySent()
     }
+
+    func testLikeNotification() throws {
+        let (updatedNotificationsScreen, previousLikes) = try TabNavComponent()
+            .goToNotificationsScreen()
+            .openNotification(withSubstring: .commentNotificationString)
+            .getNumberOfLikesForNotification()
+
+        updatedNotificationsScreen
```

Comment:
though the chain looks "broken", no additional call is made here. this is so we can use `previousLikes` value on the last verification point